calico-cloud/visibility/elastic/archive-storage.mdx

```diff
@@ -24,6 +24,10 @@ This how-to guide uses the following {{prodname}} features:
 
 ## How to
 
+:::note
+Because log files are integral to {{prodname}} diagnostics, there is no mechanism to tune down the verbosity of Kubernetes/log files. You need to filter logs using your SIEM.
```

Comment:
Not clear if it is Kubernetes log files
or
Kubernetes files/log files.

Reply:
Clarified text.